ossm/ci/post-submit.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -exo pipefail
+
+DIR=$(cd "$(dirname "$0")" ; pwd -P)
+
+GCS_PROJECT=${GCS_PROJECT:-maistra-prow-testing}
+ARTIFACTS_GCS_PATH=${ARTIFACTS_GCS_PATH:-gs://maistra-prow-testing/ztunnel}
+
+gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
+gcloud config set project "${GCS_PROJECT}"
+
+# Copy artifacts to GCS
+SHA="$(git rev-parse --verify HEAD)"
+
+if [[ "$(uname -m)" == "aarch64" ]]; then
+  ARCH_SUFFIX="-arm64"
+else
+  ARCH_SUFFIX=""
+fi
+
+tar czf ./out/rust/release/ztunnel.tar.gz ./out/rust/release/ztunnel
+gsutil cp bazel-bin/envoy_tar.tar.gz "${ARTIFACTS_GCS_PATH}/ztunnel-alpha-${SHA}${ARCH_SUFFIX}.tar.gz"
```

Comment:
1) we should copy ztunnel, not envoy binary
2) are you sure we need to rename it to ztunnel-alpha? I'd check at https://github.com/istio/istio/blob/master/bin/build_ztunnel.sh#L127-L133

Take a look at those Istio scripts to see if it's really necessary to upload a binary artifact, or if Istio is capable of building ztunnel from source. Proxy is done that way because building proxy is really an expensive task.

Reply:
> we should copy ztunnel, not envoy binary

oh, good catch! 

> Take a look at those Istio scripts to see if it's really necessary to upload a binary artifact, or if Istio is capable of building ztunnel from source. Proxy is done that way because building proxy is really an expensive task.

Good point, will do that.
Thanks.